src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,42 +1,92 @@
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import moment from 'moment';
+
 import { focusElement } from '../../../common/utils/helpers';
 import OMBInfo from '../../../common/components/OMBInfo';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
-
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
 import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { submitIntentToFile } from '../../../common/schemaform/actions';
+
+import formConfig from '../config/form';
 
 class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      verify: false
+    };
+  }
+
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  handleLoadPrefill = () => {
+    // TODO: determine payload
+    submitIntentToFile('name/ssn/other', formConfig.intentToFileUrl, formConfig.trackingPrefix);
+    // TODO: store confirmation number in formData
+  }
+
   render() {
```

Comment:
This is a pretty gnarly render statement. Is there any way we can break it down and make it easier to read?

Reply:
Good idea, `hasSavedForms` now contains the `savedForm` logic.